### DISCUSSION TITLE
Welcome to hexo-theme-amazing Discussions!

### DISCUSSION THREAD

**Discussion by removeif** [removeif/hexo-theme-amazing#51] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by removeif** [comment]
有什么问题直接在这里阐述哦！

**Comment by luovip** [comment]
博主你好，我请教下哈  1.主题首页下面的分页是超过固定文章数量会自动显示分页还是要配置某个参数    2.我用typora写的技术资料传上去后，文章中的代码部分不能像你的示例博客那样显示为黑色且显示代码颜色，这个要如何设置     3.首页下拉时可以固定左边和右边不一起向下滑动不   谢谢

**Comment by luovip** [comment]
全局博客页面各个位置的字体可以设置不，在哪里设置
